returnn/util/file_cache.py

```diff
@@ -54,9 +54,10 @@ def __init__(
         self,
         *,
         cache_directory: str = "$TMPDIR/$USER/returnn/file_cache",
-        cleanup_files_always_older_than_days: float = 31.0,
-        cleanup_files_wanted_older_than_days: float = 7.0,
+        cleanup_files_always_older_than_days: float = 7.0,
```

Comment:
I think I would keep the existing default for `cleanup_files_always_older_than_days`.
```suggestion
        cleanup_files_always_older_than_days: float = 31.0,
```
If we have otherwise enough space (and all the other logic should be responsible for that), there is no real reason to delete it. Maybe it becomes useful. 31 days is also somewhat arbitrary but I thought it to not be too long but also not too short (from own experience, how long I could maybe leave some data untouched after I would use it again).

Note that there is also tmpreaper running anyway. I'm not sure what settings we use there.

Reply:
I'm fine with this.